RELEASES.md

```diff
@@ -289,7 +289,7 @@ new version of the service.
 
 If you find that an error code that is required by your application is not
 well-documented in the protobuf service description or tested explicitly,
-please file and issue and we will clarify.
+please file and issue, and we will clarify.
```

Comment:
your tool didn't find an actual grammar problem here :) s/and issue/an issue/ ; please update to "please file an issue"; no need for a comma now as it is one phrase.

Reply:
@estesp Thank you very much. It has been modified